CONTRIBUTING.md

```diff
@@ -198,7 +198,7 @@ Please follow these formatting guidelines:
 #### Editor / IDE Support
 
 IntelliJ IDEs can
-[import](https://blog.jetbrains.com/idea/2014/01/intellij-idea-13-importing-code-formatter-settings-from-eclipse/)
+[import](https://blog.jetbrains.com/idea/2014/01/intellij-idea-`13`-importing-code-formatter-settings-from-eclipse/)
```

Comment:
are these backticks correct?

Reply:
Good catch. I have no idea where those came from. I assume IntelliJ was trying to "help" me.